src/cljs/harja/asiakas/kommunikaatio.cljs

```diff
@@ -39,7 +39,7 @@
   "Tarkastaa onko vastaus tyhjä, sisältääkö se :failure, :virhe, tai :error avaimen, tai on EiOikeutta viesti"
   [vastaus]
   (or (nil? vastaus)
-      (roolit/ei-oikeutta? vastaus)
+      (roolit/ei-oikeutta? (:response vastaus))
```

Comment:
Tässä oli ilmeisesti virhe? Jos tulee ei-oikeutta vastaus, se tulee näin:
{:status 403, :status-text "Forbidden", :failure :error, :response #harja.domain.roolit.EiOikeutta{:syy "lol"}}

Eli instanssi pitää etsiä :response avaimesta